addons/mail/models/mail_activity_plan_template.py

```diff
@@ -23,6 +25,21 @@ class MailActivityPlanTemplate(models.Model):
         domain="['|', ('res_model', '=', False), '&', ('res_model', '!=', False), ('res_model', '=', parent.res_model)]",
         ondelete='restrict', required=True
     )
+    # Activity type delay fields are ignored in favor of these
+    delay_count = fields.Integer(
```

Comment:
Shouldn't we pre-populate with values coming from the activity type ? then people can change it if they want but that way activity type act as default values and can be configured once